### PR TITLE
Eli changes + fixes from comments

### PR DESCRIPTION
**Pull request by jonasfj**
@eliperelman, due to a bug in github (reported) I can't easily edit when you push the branch to your repo.

Anyways, here is a PR with comment addressed, I think...

Please review, I did this copy/paste trick rather quickly.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,11 +14,6 @@ import (
 
 var debug = Debug("livelog")
 
-const (
-	DEFAULT_PUT_PORT = 60022
-	DEFAULT_GET_PORT = 60023
-)
-
```
Comment (petemoore) [inline main.go]: we need this - please don't delete

File: main.go [hunk 2]
```diff
@@ -34,7 +28,7 @@ func abort(writer http.ResponseWriter) error {
 	return nil
 }
 
-func startLogServe(stream *stream.Stream, getAddr string) {
+func startLogServe(stream *stream.Stream) {
```
Comment (petemoore) [inline main.go]: needed - please don't delete here

File: main.go [hunk 3]
```diff
@@ -47,17 +41,23 @@ func startLogServe(stream *stream.Stream, getAddr string) {
...
+			w.WriteHeader(405)
+			fmt.Fprint(w, "method not allowed")
 		} else {
 			getLog(stream, w, r)
 		}
 	})
 
 	server := http.Server{
-		Addr:    getAddr,
+		Addr:    ":60023",
```
Comment (petemoore) [inline main.go]: needed - please don't revert

File: main.go [hunk 4]
```diff
@@ -145,33 +152,9 @@ func main() {
...
-				os.Exit(outOfRangeExitCode)
-			}
-			addr = ":" + port
-		}
-		return
-	}
-
-	putAddr := portAddressOrExit("LIVELOG_PUT_PORT", DEFAULT_PUT_PORT, 64, 65)
-	getAddr := portAddressOrExit("LIVELOG_GET_PORT", DEFAULT_GET_PORT, 66, 67)
-
```
Comment (petemoore) [inline main.go]: All needed - please don't delete

File: main.go [hunk 5]
```diff
@@ -145,33 +152,9 @@ func main() {
...
 	server := http.Server{
 		// Main put server listens on the public root for the worker.
-		Addr:    putAddr,
+		Addr:    ":60022",
```
Comment (petemoore) [inline main.go]: needed - please don't revert

File: main.go [hunk 6]
```diff
@@ -217,7 +200,7 @@ func main() {
 
 		// Initialize the sub server in another go routine...
 		debug("Begin consuming...")
-		go startLogServe(stream, getAddr)
+		go startLogServe(stream)
```
Comment (petemoore) [inline main.go]: needed - please don't revert

**Comment by jonasfj** [comment]
hmm... how did it decide to undo all that... wrong base I guess... that's no good..

**Pull request closed without merging**